source/language/types.rst

```diff
@@ -267,6 +267,38 @@ are all of type ``float[64]``.
 
 Note that `e` is a valid identifier. `e/E` are also used in scientific notation where appropriate.
 
+Mathematical functions available for constant initialization
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+In addition to simple arithmetic functions used in expressions initializing constants,
+OpenQASM 3 offers the following built-in mathematical operators followed by
+their argument expression in parentheses:
+
+.. container::
+   :name: tab:built-in-math
+
+   .. table:: Built-in mathematical functions in OpenQASM3.
+
+      +--------+
+      | arccos |
+      +--------+
+      | arcsin |
+      +--------+
+      | arctan |
+      +--------+
+      | cos    |
```

Comment:
For the trig functions, I'd expect the allowed overloads to be

- `cos(float) -> float`
- `cos(angle) -> float` (danger)

and similar for other functions.  We should specify that `cos(float)` etc take their angle in radians.  _However_, it's not at all clear to me what the precision of the output `float` should be in the potential `cos(angle) -> float` overload; I'd expect the function to work, but I'm not sure our current type system can support it.  Perhaps best to omit it for now.